cli/command/cli.go

```diff
@@ -280,7 +286,7 @@ func newAPIClientFromEndpoint(ep docker.Endpoint, configFile *configfile.ConfigF
 }
 
 func resolveDockerEndpoint(s store.Store, contextName string, opts *cliflags.CommonOptions) (docker.Endpoint, error) {
-	if contextName != "" {
+	if contextName != "" || contextName == DefaultContextName {
```

Comment:
feels like the comparison here is wrong. should be:
```suggestion
	if contextName != "" && contextName != DefaultContextName {
```